src/pages/Flow/Settings/ClockForm.vue

```diff
@@ -154,3 +158,10 @@ export default {
     </div>
   </v-container>
 </template>
+
+<style>
+/* stylelint-disable */
+.v-autocomplete__content.v-menu__content .v-select-list {
+  max-width: 100%;
+}
+</style>
```

Comment:
Not sure if we should keep this or target it through a parent class? I think it could also fix [413](https://github.com/PrefectHQ/ui/issues/413)?

Reply:
Oh yeah that's a good flag, if we can target this through a parent class that would be good. You'll probably need to add the class through the `v-autocomplete` component though, with the `classList` option param

Reply:
(the v-select gets attached to the body node so it's not scope-able within the component)

Reply:
I was playing around in the inspector and tried adding this in the console `document.body.getElementByClassName('v-autocomplete__content')[0].classList.add('tz-autocomplete')` then tried styling it through the newly added class but it didn't apply. Though, I think I'm not targeting it correctly.

When you say that v-select gets attached to the body node, so the class that styles the content is outside of the component so thats why we can't just give it a parent class through `class=""`?